spec/models/candidate_interface/personal_details_form_spec.rb

```diff
@@ -1,6 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe CandidateInterface::PersonalDetailsForm, type: :model do
+  let(:data) do
+    {
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      date_of_birth: Faker::Date.birthday,
+      first_nationality: NATIONALITIES.sample,
+      second_nationality: NATIONALITIES.sample,
+      english_main_language: %w[yes no].sample,
```

Comment:
a tiny helper would be great!

Reply:
Ehh it would make it easier if this ever changes but I'm not sure it's necessary here, this is quite a small bit of functionality... 🤷‍♂